catalog/core/catalog-core-standardframework/src/main/java/ddf/catalog/resource/download/ReliableResourceCallable.java

```diff
@@ -25,19 +25,20 @@
 import com.google.common.io.CountingOutputStream;
 
 /**
- * ReliableResourceCallable is responsible for reading product data from its @InputStream and then writing that data
- * to a @FileBackedOutputStream (that will be concurrently read by a client), and optionally caching the product to 
- * the file system. It is a @Callable that is started via a @Future by the @ReliableResourceDownloadManager class.
- *
- * The client uses the @ReliableResourceInputStream to read from the @FileBackedOutputStream. 
- *
- * This class will read bytes in chunks (whose size is specified by the caller) until it either reaches the EOF or it 
- * is interrupted (either by an @IOException or the @CachedResource).
- *
- * If the @InputStream is read until the EOF, then a -1 is returned indicating the entire stream was read successfully. 
- * Otherwise, the number of bytes read thus far is returned so that the caller can react accordingly (e.g., reopen the 
- * product @InputStream and skip forward that many bytes already read).
- *
+ * ReliableResourceCallable is responsible for reading product data from its @InputStream and then
+ * writing that data to a @FileBackedOutputStream (that will be concurrently read by a client), and
+ * optionally caching the product to the file system. It is a @Callable that is started via a @Future
+ * by the @ReliableResourceDownloadManager class.
+ * <p>
+ * The client uses the @ReliableResourceInputStream to read from the @FileBackedOutputStream.
+ * <p>
+ * This class will read bytes in chunks (whose size is specified by the caller) until it either
+ * reaches the EOF or it is interrupted (either by an @IOException or the @CachedResource).
+ * <p>
+ * If the @InputStream is read until the EOF, then a -1 is returned indicating the entire stream was
```

Comment:
Not sure this part of the comment belongs here and is still accurate. We should either remote it, or move it to the appropriate method and make sure it is accurate.